.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,27 @@
+name: Deploy
+on:
+  push:
```

Comment:
if you want to submit this please remove this since it will deploy to staging on any push to the repo!